tests/components/withings/test_common.py

```diff
@@ -163,8 +163,9 @@ async def test_data_manager_webhook_subscription(
         WebhookConfig(id="1234", url="http://localhost/api/webhook/1234", enabled=True),
     )
 
-    # pylint: disable=protected-access
+    # pylint: disable-next=protected-access
     data_manager._notify_subscribe_delay = datetime.timedelta(seconds=0)
+    # pylint: disable-next=protected-access
     data_manager._notify_unsubscribe_delay = datetime.timedelta(seconds=0)
```

Comment:
Another case where I'm not sure it's really necessary to use `disable-next` for `protected-access`.

Reply:
Combined.